Applications/WebPortal/WebApp/library/source/pages/account_page/compliance_page/symbol_input_field.tsx

```diff
@@ -0,0 +1,39 @@
+import { css, StyleSheet } from 'aphrodite';
```

Comment:
Maybe want to remove this from the pull request unless you really need it.

Reply:
The field imports or the file?

Reply:
File